### ISSUE TITLE
Switching spaces

### ISSUE THREAD

**Issue by 24unix** [cmacrae/spacebar#33]
Displaying the spaces works great, would it be possible to make the numbers react on mouse clicks for switching spaces?

**Comment by cmacrae** [comment]
Hey @24unix 👋 Sorry for the delayed response on this. It seems I wasn't getting notifications for a while and I've been otherwise occupied.

Nice idea for a feature! I'll see what I can do :) PRs are always welcome, though!

**Comment by emninger** [comment]
I agree with @24unix  - a native solution to make the numbers clickable would be great. As a workaround just a question (i'm not so familiar with MacOS): Is there something for the Mac display server like _xdotools_ for X11?

**Comment by junwei-wang** [comment]
I find it is handy to just use default key provided by the system to switch workspaces. I mean by "CTR+n", you can switch to workspace n.

**Comment by nilsi** [comment]
Would also like to mouse click to switch to a higher number and spaces I don't access very often. I got 8 different spaces all just having icons so it would sometimes be handy to use the mouse.

How could this be implemented? Via the Accessibility API or something else. I guess yabai is doing this when running "yabai -m space --focus 4" for example.

**Comment by 24unix** [comment]
@junwei-wang ctrl-x doesn't play nicely with multi displays setup. You could add identical shortcuts to different displays, but are limited to 18 spaces. Bad with three displays. I found a workaround with Total-Spaces, it combines spaces to displays, but still the option to click on a number in spacebar would be fine. Actually, Total Spaces offers that in a menuitem, but spacebar is more present.

**Comment by junwei-wang** [comment]
> @junwei-wang ctrl-x doesn't play nicely with multi displays setup. You could add identical shortcuts to different displays, but are limited to 18 spaces. Bad with three displays. I found a workaround with Total-Spaces, it combines spaces to displays, but still the option to click on a number in spacebar would be fine. Actually, Total Spaces offers that in a menuitem, but spacebar is more present.

Make sense.

**Comment by 24unix** [comment]
Any updates on this one?
Since TotalSpaces doesn't work under Monterey my current setup is quite broken :(
